user-interface/src/data-verification/ConsolidationOrderAccordion.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import { orderType, orderStatusType } from '@/lib/utils/labels';
+import { BrowserRouter } from 'react-router-dom';
+import { ConsolidationOrder } from '@common/cams/orders';
+import {
+  ConsolidationOrderAccordion,
+  ConsolidationOrderAccordionProps,
+} from '@/data-verification/ConsolidationOrderAccordion';
+import { MockData } from '@common/cams/test-utilities/mock-data';
+import { OfficeDetails } from '@common/cams/courts';
+import { formatDate } from '@/lib/utils/datetime';
+import * as FeatureFlagHook from '@/lib/hooks/UseFeatureFlags';
+
+vi.mock(
+  '../lib/components/SearchableSelect',
+  () => import('../lib/components/SearchableSelect.mock'),
+);
+
+function findAccordionHeading(id: string) {
+  const heading = screen.getByTestId(`accordion-heading-${id}`);
+  expect(heading).toBeInTheDocument();
+  expect(heading).toBeVisible();
+  return heading;
+}
+
+function findAccordionContent(id: string, visible: boolean) {
+  const content = screen.getByTestId(`accordion-content-${id}`);
+  expect(content).toBeInTheDocument();
+  if (visible) {
+    expect(content).toBeVisible();
+  } else {
+    expect(content).not.toBeVisible();
+  }
+  return content;
+}
+
+describe('ConsolidationOrderAccordion tests', () => {
+  const order: ConsolidationOrder = MockData.getConsolidationOrder();
+  const offices: OfficeDetails[] = MockData.getOffices();
+  const regionMap = new Map();
+
+  function renderWithProps(props?: Partial<ConsolidationOrderAccordionProps>) {
+    const defaultProps: ConsolidationOrderAccordionProps = {
+      order,
+      officesList: offices,
+      orderType,
+      statusType: orderStatusType,
+      onOrderUpdate: () => {},
+      onExpand: () => {},
+      regionsMap: regionMap,
+    };
+
+    const renderProps = { ...defaultProps, ...props };
+    render(
+      <BrowserRouter>
+        <ConsolidationOrderAccordion {...renderProps} />
+      </BrowserRouter>,
+    );
+  }
+
+  test('should render an order', async () => {
+    renderWithProps();
+
+    const heading = findAccordionHeading(order.id!);
+    expect(heading?.textContent).toContain(order.courtName);
+    expect(heading?.textContent).toContain(formatDate(order.orderDate));
+
+    const content = findAccordionContent(order.id!, false);
+
+    order.childCases.forEach((childCase) => {
+      expect(content?.textContent).toContain(childCase.caseTitle);
+      expect(content?.textContent).toContain(formatDate(childCase.dateFiled));
+      childCase.docketEntries.forEach((de) => {
+        expect(content?.textContent).toContain(de.summaryText);
+        expect(content?.textContent).toContain(de.fullText);
+      });
+    });
+  });
+
+  test('should display pending order properly', () => {});
+
+  test('should display approved order properly', () => {});
+
+  test('should display rejected order properly', () => {});
```

Comment:
If we've already covered these tests, lets remove the stubs or implement them if not.

Reply:
We are leaving the rejection stub for now. The other two were implemented.